lib/govuk_app_config/govuk_error/configure.rb

```diff
@@ -1,10 +1,4 @@
 GovukError.configure do |config|
-  config.before_send = proc { |e|
-    GovukStatsd.increment("errors_occurred")
```

Comment:
Looks like this never had a test, which is unfortunate. Would it be possible to add one now that we've moved the code? Would be great to have a test that confirms that these statsd values are incremented when an error occurs.